tests/settings.rs

```diff
@@ -1,11 +1,113 @@
 extern crate ron;
 extern crate vangers;
+extern crate speculate;
 
-use std::fs::File;
+use speculate::speculate;
 
-#[test]
-fn load_settings() {
-    let file = File::open("config/settings.template.ron").unwrap();
-    ron::de::from_reader::<_, vangers::config::settings::Settings>(file)
-        .unwrap();
+speculate! {
+    describe "settings" {
+        before {
+            let _default_settings = r#"
```

Comment:
Why are you using underscore in front of the variable name? You should use underscores only for unused variables

Reply:
@vitvakatu Rust compiler asked me to do this.